wsgi.py

```diff
@@ -102,7 +102,10 @@ def post_publish():
         )
     }
 
-    headers = {'x-rh-insights-request-id': data_id}
+    b64_identity = request.headers['x-rh-identity']
```

Comment:
I would prefer `dict.get` here. Rather safe than sorry. :wink:

Reply:
Ok, agreed! 
Sorry... I did these changes 2 days ago, and should have applied `.get` here like we did elsewhere

Reply:
done